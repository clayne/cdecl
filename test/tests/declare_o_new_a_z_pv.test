cdecl @ @ -xc++ @ declare new[] as operator (n as std::size_t) returning pointer to void @ 0
