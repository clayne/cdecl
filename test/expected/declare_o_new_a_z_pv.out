void* operator new[](std::size_t n);
